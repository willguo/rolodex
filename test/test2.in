Pesche, Mueller, INVALID, 294 282 4920, rainbow
Rita Dvorak, emerald green, 12345, 292 383 2729
Rita D'vorak, black, 23222, 121 234 1231
Jamieson Johnston, gray, 38410, 628 102 3672
James Johnston, gray, 38410, 628 102 3672
James K. Johnston, gray, 38410, 628 102 3672
Dueber, Nicholas, (666)-666-6666, cyan, 94704
123, 123, 123, 123, 123
234234234