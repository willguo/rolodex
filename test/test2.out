{
  "entries": [
    {
      "color": "black", 
      "firstname": "Rita", 
      "lastname": "D'vorak", 
      "phonenumber": "121-234-1231", 
      "zipcode": "23222"
    }, 
    {
      "color": "cyan", 
      "firstname": "Nicholas", 
      "lastname": "Dueber", 
      "phonenumber": "666-666-6666", 
      "zipcode": "94704"
    }, 
    {
      "color": "emerald green", 
      "firstname": "Rita", 
      "lastname": "Dvorak", 
      "phonenumber": "292-383-2729", 
      "zipcode": "12345"
    }, 
    {
      "color": "gray", 
      "firstname": "James", 
      "lastname": "Johnston", 
      "phonenumber": "628-102-3672", 
      "zipcode": "38410"
    }, 
    {
      "color": "gray", 
      "firstname": "James K.", 
      "lastname": "Johnston", 
      "phonenumber": "628-102-3672", 
      "zipcode": "38410"
    }, 
    {
      "color": "gray", 
      "firstname": "Jamieson", 
      "lastname": "Johnston", 
      "phonenumber": "628-102-3672", 
      "zipcode": "38410"
    }
  ], 
  "errors": [
    0, 
    7, 
    8
  ]
}